src/config/config.js

```diff
@@ -18,7 +18,7 @@
 const config = {                                                        //remote api
   //apiUrl: 'http://localhost/api/',
   apiUrl: 'https://te.loc/api/',
-  streamingUrl: 'http://localhost/',
+  streamingUrl: 'http://te.loc/',
```

Comment:
can you just keep the streamingUrl : 'http://localhost/ in the config file and commented out, just like I did with the api? just push that and then you're good and can merge it. i'm approving it

Reply:
if you leave the streaming url as localhost the sound is not going to load on the take cards... so when I pulled and run the first time i did not see the waves loading and I thought it was a bug, spend some time debugging and then I went to the config file and I saw that the streamingUrl was pointing to localhost... we need to have both pointing to localhost or te.loc, or is there another reason to have them pointing to different directions?

Reply:
no it is meant to be pointing to the same place but the reason i wanted it commented out is because it's something that we are using for local development. we are not always going to be working with the te.loc database so we should have localhost in there. underneath in the config we have the configuration for te.loc. when we want to work with the te.loc api then we can just uncomment all the `http://te.loc/` addresses. 
it's also just good to keep the configurations separate from each other.

Reply:
that is my fault. After your push on it now we should just leave our config files out of commits in the future, so whatever changes we have just stay local.

Reply:
no problem it is both fault because I should have catch that haha anyway I just pushed